src/dlgTriggerEditor.cpp

```diff
@@ -5250,7 +5250,7 @@ void dlgTriggerEditor::fillout_form()
                 }
                 else
                 {
-                    if( pT->isActive() )
+                    if( script->isActive() )
```

Comment:
Another code block defect, remove the lines current numbered "after" 5247 to 5250 inclusive.